src/main/java/com/vinylteam/vinyl/web/templater/PageGenerator.java

```diff
@@ -52,6 +53,13 @@ public void process(String fileName, List<UniqueVinyl> list, Map<String, String>
         process(fileName, list, new ArrayList<>(), attributes, writer);
     }
 
+    public void processStores(String fileName, List<Shop> list, Map<String, String> attributes, Writer writer) {
```

Comment:
It should be overridden method process(...), like others from line 35 to 54.

Reply:
Did you mean overloaded I suppose?